.github/workflows/data-prepper-trace-analytics-raw-span-e2e-tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         java: [11, 17, 21, docker]
-        otelVersion: ['0.9.0-alpha', '0.16.0-alpha']
+        otelVersion: ['1.3.2-alpha']
```

Comment:
Can we keep an old version in here and still have tests pass? I'd expect this to be ok since the wire format is unchanged.

```
otelVersion: ['0.16.0-alpha', '1.3.2-alpha']
```

Reply:
@dlvenable I do not think that's possible.

Reply:
Why not? The wire format is the same is it not?

Reply:
Is this the OTEL version on the client side? If yes, that should work.

Reply:
I have verified this locally and it should work.